app/json/slate_configs.json

```
@@ -89,6 +89,74 @@
       }
     ]
   },
+  {
+    "id": "f99178fb-6bd0-4fa1-8109-cda181b697f6",
+    "displayName": "New Tab Recommendations",
```

Matt is correct. Firefox supplies these strings internally.

The string that's used by Firefox today is this:
```suggestion
    "displayName": "Recommended by Pocket",
```